backend/internal/controller/server/msg_handler.go

```diff
@@ -151,5 +146,59 @@ func writeHTTPError(w http.ResponseWriter, header int, msg string) error {
 }
 
 func isValidTeamID(teamID int) bool {
-	return (teamID == model.TeamWhite || teamID == model.TeamBlue)
+	return (teamID == config.TeamWhite || teamID == config.TeamBlue)
+}
+
+func (s server) ShotParametersHandler(w http.ResponseWriter, r *http.Request) {
+	var upgrader websocket.Upgrader
+	c, err := upgrader.Upgrade(w, r, nil)
+	if err != nil {
+		log.Error(err)
+	}
+
+	defer c.Close()
+
+	for {
+		_, receivedMsg, err := c.NextReader()
+		if err != nil {
+			log.Error(err)
+			continue
+		}
+
+		shot, err := adapter.UnpackShotMsg(receivedMsg)
+		if err != nil {
+			log.Error(err)
+		}
+
+		err = s.game.UpdateShotsData(shot)
+		if err != nil {
+			log.Error(err)
+		}
```

Comment:
Error handling

You cannot use `shot` if there was an error in unpacking?

Reply:
We will fix it.